pkg/aws/ec2/instance.go

```diff
@@ -110,8 +115,16 @@ func (i *ec2Instance) LoadDetails(ec2APIHelper api.EC2APIHelper) error {
 			i.instanceSubnetID, i.instanceID)
 	}
 	i.instanceSubnetCidrBlock = *instanceSubnet.CidrBlock
-
 	i.subnetMask = strings.Split(i.instanceSubnetCidrBlock, "/")[1]
+	// Cache IPv6 CIDR block if one is present
+	for _, v6CidrBlock := range instanceSubnet.Ipv6CidrBlockAssociationSet {
+		if v6CidrBlock.Ipv6CidrBlock != nil {
+			i.instanceSubnetV6CidrBlock = *v6CidrBlock.Ipv6CidrBlock
+			i.subnetV6Mask = strings.Split(i.instanceSubnetV6CidrBlock, "/")[1]
```

Comment:
Is instanceSubnetV6CidrBlock guaranteed to be split-able to at least two parts? Want to be make sure we won't have a panic in possible v6-not-supported cases.